src/main/kotlin/graphics/scenery/backends/vulkan/VulkanRenderer.kt

```diff
@@ -1516,6 +1735,14 @@ open class VulkanRenderer(hub: Hub,
         if(hub?.getWorkingHMDDisplay()?.hasCompositor() == true) {
             hub?.getWorkingHMDDisplay()?.wantsVR(settings)?.update()
         }
+        
+        postRenderLambdas.forEach {
+            it.invoke()
+        }
+
+        postRenderLambdas.forEach {
```

Comment:
All `postRenderLambdas` get invoked twice, this is likely a merge mishap and should be gotten rid of.